src/components/TopicPage/ArticleList.jsx

```diff
@@ -0,0 +1,22 @@
+import { useEffect, useState } from "react";
+import { getArticles } from "../../api";
+import PageBar from "./PageBar";
+
+function ArticleList({ articleList }) {
+  return (
+    <>
+      <ul key="articleList">
+        {articleList.map((article) => {
+          return (
+            <div key={article.article_id}>
+              <p>{article.title}</p>
+              <img src={article.article_img_url} alt="article image" />
+            </div>
```

Comment:
You have a lot of information from the object that can be used here. It's your choice but it would be nice to see things like the topic, author and created at. This will help show your sorting and filtering features when you get to that ticket 😄